outputs/aws/s3/s3.go

```diff
@@ -0,0 +1,119 @@
+package s3
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	awssdk "github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+
+	aws "github.com/falco-talon/falco-talon/internal/aws/client"
+	"github.com/falco-talon/falco-talon/internal/rules"
+	"github.com/falco-talon/falco-talon/outputs/model"
+	"github.com/falco-talon/falco-talon/utils"
+)
+
+type Config struct {
+	Bucket string `mapstructure:"bucket" validate:"required"`
+	Prefix string `mapstructure:"prefix" validate:"required"`
+	Region string `mapstructure:"region" validate:"omitempty,oneof=us-east-2 us-east-1 us-west-1 us-west-2 af-south-1 ap-east-1 ap-south-2 ap-southeast-3 ap-southeast-4 ap-south-1 ap-northeast-3 ap-northeast-2 ap-southeast-1 ap-southeast-2 ap-northeast-1 ca-central-1 ca-west-1 eu-central-1 eu-west-1 eu-west-2 eu-south-1 eu-west-3 eu-south-2 eu-north-1 eu-central-2 il-central-1 me-south-1 me-central-1 sa-east-1"`
```

Comment:
For aws.client, the validation is made automatically when we create the session, here's an error example:

```
2024-06-18T18:12:43+02:00 ERR init error="operation error STS: GetCallerIdentity, https response error StatusCode: 0, RequestID: , request send failed, Post \"https://sts.eu-west-13.amazonaws.com/\": dial tcp: lookup sts.eu-west-13.amazonaws.com: no such host" actionner_category=aws
```

This region parameter is there because the bucket S3 can be in another region than the created session.

Reply:
It makes sense, it allows some use cases that I think it's worth keeping it here. 

But I still don't enjoy having the list of regions hard coded here, prefer to fail when calling than needing to maintain this in the long term, WDYT?

Reply:
Also, can we have the defaultRegion set the same as session instead of hard-coding us-east-1? I guess we'll face similar issues with other services integrations that only offer global endpoints.